Aptoide/aptoide-client-v8
# [MOB-648] Remove Birthdate and Newsletter

**What does this PR do?**

   This PR aims at removing all info related to birthdate and newsletter as we are not going to store that information anymore.

**Database changed?**

 No

**Where should the reviewer start?**

- [ ] ManageUserPresenter.java

**How should this be manually tested?**

This was removal of code. You could check if nothing breaks but you can not signup at the moment, so maybe try through the edit profile? Eventhough that could be kind of pointless because we are redoing those views.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-648](https://aptoide.atlassian.net/browse/MOB-648)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
